packages/web-components/src/donut-chart/donut-chart.options.ts

```diff
@@ -36,5 +36,5 @@ export interface IChartProps {
   /**
    * data for the points in the chart
    */
-  chartData?: IChartDataPoint[];
+  chartData?: ChartDataPoint[];
```

Comment:
make this change for horizontal bar chart as well

Reply:
done